app/sentinel/logger/logmethods.go

```diff
@@ -0,0 +1,143 @@
+package logger
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+)
+
+// Level represents log levels.
+type Level int
+
+// Define log levels as constants.
+const (
+	Off Level = iota
+	Fatal
+	Error
+	Warn
+	Info
+	Audit
+	Debug
+	Trace
+)
+
+var (
+	currentLevel Level        // Holds the current log level.
+	mux          sync.RWMutex // Protects access to currentLevel.
+)
+
+var currentTime = func() string {
+	return time.Now().Local().Format(time.DateTime)
+}
+
+func init() {
```

Comment:
Might be better to explicitly call an `Initialize()` function instead of implicitly igniting here… not sure… need to think about it.